doc/examples/plot_line_hough_transform.py

```diff
@@ -64,70 +64,71 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
-# Construct test image
-
+# Constructing test image.
 image = np.zeros((100, 100))
-
-
-# Classic straight-line Hough transform
-
 idx = np.arange(25, 75)
 image[idx[::-1], idx] = 255
 image[idx, idx] = 255
 
+# Classic straight-line Hough transform.
 h, theta, d = hough_line(image)
 
-fig, (ax1, ax2, ax3) = plt.subplots(1, 3, figsize=(8,4))
+# Generating figure 1.
+fig, (ax1, ax2, ax3) = plt.subplots(1, 3, figsize=(12, 6))
+plt.tight_layout()
 
-ax1.imshow(image, cmap=plt.cm.gray)
+ax1.imshow(image, cmap=cm.gray)
 ax1.set_title('Input image')
 ax1.set_axis_off()
 
-ax2.imshow(np.log(1 + h),
-             extent=[np.rad2deg(theta[-1]), np.rad2deg(theta[0]),
-                     d[-1], d[0]],
-             cmap=plt.cm.gray, aspect=1/1.5)
+ax2.imshow(np.log(1 + h), extent=[np.rad2deg(theta[-1]), np.rad2deg(theta[0]),
+           d[-1], d[0]], cmap=cm.gray, aspect=1/1.5)
 ax2.set_title('Hough transform')
 ax2.set_xlabel('Angles (degrees)')
 ax2.set_ylabel('Distance (pixels)')
 ax2.axis('image')
 
-ax3.imshow(image, cmap=plt.cm.gray)
-rows, cols = image.shape
+ax3.imshow(image, cmap=cm.gray)
+row1, col1 = np.shape(image)
```

Comment:
I think np.shape(image) is more clear, but you have a point. I'll change it back.

`rows, cols` assumes two values in this example, each one used for a image set. In order to adjust the image size, I stablished two variable sets: `row1, col1` and `row2, col2`.